docs/docs/how-it-works/01_flex-config.md

```diff
@@ -6,18 +6,24 @@ title: flex-config
 
 This package manages a json artifact that can be used to version configuration elements on a per-twilio-account basis. We can think of this as allowing us to configure, dev, qa, test, production or any other environments individually. This configuration relates specifically to the the configuration for flex discussed [here](https://www.twilio.com/docs/flex/developer/ui/configuration) and works by injecting the custom object into ui_attributes within the flex configuration object. The plugin is then able to reference these variables. The first example being, hosting the domain name of the associated serverless-functions.
 
-_NOTE_ changes are deep merged with whatever exists on the environment so existing configuration is preserved unless otherwise overridden by whats in version control.
-
 ---
 
-This is a simple set of scripts that help deploy Flex configuration settings. Currently it only supports setting `ui_attributes` properties and `taskrouter_skills` within the [Flex UI Configuration](https://www.twilio.com/docs/flex/developer/ui/configuration).
+The scripts work by merging the common attributes with the environment specifc attributes and then deploying them as part of the release pipeline.
+
+_NOTE_ there are two mechanisms for managing the flex-config.
+
+1. manging the files via the [administration panel](/feature-library/flex-v2/admin-ui).  This is the default mechanism and as a result a deploy from the pipeline will only add new elements, it will not overwrite existing configuration.
+2.  manging completely through what is in version control.  In which case, you need to create an environment *variable* in your github actions configuration `OVERWRITE_CONFIG` = `true`
+
+Currently it only supports managing `ui_attributes` properties and `taskrouter_skills` within the [Flex UI Configuration](https://www.twilio.com/docs/flex/developer/ui/configuration).
 
 - `ui_attributes` managed per environment
 - `taskrouter_skills` managed across all environments
 
-The version controlled configuration is merged with the configuration already existing in the environment.
 
-## To Use Locally
+## To Use Locally (typically not neccessary)
+
+_NOTE_ although not typically neccessary, these instructions are intended for completeness and only for application of config on a personal development environment, not a shared development environment.  For shared environments please use the github release pipeline scripts.
```

Comment:
It might be clearer to note that when running locally, appConfig.js overrides flex-config and should be used.